com.termius.termius.yml

```diff
@@ -0,0 +1,73 @@
+app-id: com.termius.termius
+
+runtime: org.freedesktop.Platform
+runtime-version: "18.08"
+sdk: org.freedesktop.Sdk
+
+base: org.electronjs.Electron2.BaseApp
+base-version: "18.08"
+
+command: termius-app
+
+finish-args:
+  # X11/Wayland
+  - --socket=x11
+  - --share=ipc
+  - --socket=wayland
+  - --device=dri
+  # Sync
+  - --share=network
+  # Filesystem access
+  - --filesystem=host
+  # Allow access to keyring
+  - --talk-name=org.freedesktop.secrets
+  # SSH
+  - --socket=ssh-auth
+  - --own-name=org.KDE.*
```

Comment:
This it too broad, the app obvously doesn't own everything on `org.kde`. Run the app with `flatpak run --log-session-bus com.termius.termius` and check what actual names on D-Bus it needs to talk to.

Reply:
```Gtk-Message: 22:08:46.203: Failed to load module "canberra-gtk-module"
C1: -> org.freedesktop.DBus call org.freedesktop.DBus.Hello at /org/freedesktop/DBus
C2: -> org.freedesktop.DBus fake wildcarded AddMatch for org.freedesktop.portal
C3: -> org.freedesktop.DBus fake wildcarded AddMatch for org.mpris.MediaPlayer2.com.termius.termius
C4: -> org.freedesktop.DBus fake wildcarded AddMatch for com.termius.termius
C5: -> org.freedesktop.DBus fake AddMatch for org.freedesktop.secrets
C6: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.secrets
C7: -> org.freedesktop.DBus fake wildcarded AddMatch for org.KDE
C8: -> org.freedesktop.DBus fake ListNames
B1: <- org.freedesktop.DBus return from C1
B2: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameAcquired at /org/freedesktop/DBus
B3: <- org.freedesktop.DBus return from C2
*SKIPPED*
B4: <- org.freedesktop.DBus return from C3
*SKIPPED*
B5: <- org.freedesktop.DBus return from C4
*SKIPPED*
B6: <- org.freedesktop.DBus return from C5
*SKIPPED*
B7: <- org.freedesktop.DBus return error org.freedesktop.DBus.Error.NameHasNoOwner from C6
*SKIPPED*
B8: <- org.freedesktop.DBus return from C7
*SKIPPED*
B9: <- org.freedesktop.DBus return from C8
C9: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.DBus
C10: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Documents
*SKIPPED*
B10: <- org.freedesktop.DBus return from C9
*SKIPPED*
B11: <- org.freedesktop.DBus return from C10
*SKIPPED*
C11: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
B12: <- org.freedesktop.DBus return from C11
C12: -> org.freedesktop.DBus call org.freedesktop.DBus.ListNames at /org/freedesktop/DBus
B13: <- org.freedesktop.DBus return from C12
C13: -> org.freedesktop.DBus call org.freedesktop.DBus.RemoveMatch at /org/freedesktop/DBus
B14: <- org.freedesktop.DBus return from C13```

Reply:
If it's the whole output, the app probably doesn't need anything like `org.kde.`.

Reply:
ok im try